test_scripts/Polices/External_UCS_Informing_HMI/017_ATF_N_GetListofPermissions_empty_due_to_missing_entityID_OnAppPermissionChange.lua

```diff
@@ -103,7 +104,7 @@ end
 --[[ Test ]]
 commonFunctions:newTestCasesGroup("Test")
 
-function Test:TestStep_GetListofPermissions_entityType_missing()
+function Test:TestStep_GetListofPermissions_entityID_missing()
   local RequestIdListOfPermissions = self.hmiConnection:SendRequest("SDL.GetListOfPermissions", {appID = self.applications[config.application1.registerAppInterfaceParams.appName]})
   
   EXPECT_HMIRESPONSE(RequestIdListOfPermissions, {
```

Comment:
@istoimenova please see updatet in commit ea2a524d288dd0d792998c185777040e0e8192a6